wordpress-mobile/WordPress-Android
# WP Stories : Remote image thumbnail issue

Fixes https://github.com/Automattic/portkey-android/issue/375 - [Port Key PR](https://github.com/Automattic/portkey-android/pull/387)

**NOTE: BUILDS ON TOP OF #12297** 

## Findings

When remote images are being loaded from Pexels, the thumbnail `ImageView` is not being set with the downloaded drawable. 

To rectify this issue, here are a few approaches that were tried :

1. I added a listener to the `Glide` object to see if there would be any log output but there was none. 
2. I utilized a `GlobalLayoutListener` to only apply the image when we are sure the `ImageView` was loaded but that didn't solve the problem either. 

## Solution

The solution I ended up going with was to delay the setting loading of the image by 1 second so any other operation that's causing the discrepancy would have enough time to be completed so the image loading operation is able to be completed successfully.  

## Testing

1. Create a new Story. 
2. Select upload. 
3. Choose from Free Photo Library. 
4. Select an image. 
5. Ensure that the thumbnail is loaded. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
